File: Python Programming/problems on pattern printing/problem35.py
'''
34. Accept number from user and display below pattern. 
	Input :  5  
	Output : 1 * 2 * 3 * 4 * 5 *   
	
	Author : Milind	Nalawade
	
	Date : 29/06/2021
	
	Language :Python 3.9.0

'''

#====
#Logic
#====
def Print(no):
	i=1
	while i<=no:
		print(i," * ",end=" ")
		i+=1
		
#=============
#Entry point function
#=============
def main():
	no=int(input("Enter your number : "))
	Print(no)
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C Programming/problems on pattern printing/problem35.c
/*

35. Accept number from user and display below pattern. 
	Input :  5  
	Output : 1 * 2 * 3 * 4 * 5 *   
	
	Author : Milind	Nalawade
	
	Date : 29/06/2021
	
	Language : C programming
	
*/

#include<stdio.h>

//Logic

void Print(int no)
{
	int i=1;
	while(i!=no+1)
	{
		printf(" %d *",i);
		i++;
	}
}

//Entry point function
 
int main()
{
	int no=0;
	printf("Enter your number : ");
	scanf("%d",&no);
	Print(no);
	return 0;
}	